db/migrate/20170519153350_add_showback_models.rb

```diff
@@ -0,0 +1,73 @@
+class AddShowbackModels < ActiveRecord::Migration[5.0]
+  def up
+    create_table :showback_usage_types, :id => :bigserial, :force => :cascade do |t|
+      t.string     :category, :allow_nil => false
+      t.string     :description
+      t.string     :measure, :allow_nil => false
```

Comment:
Deleting all. How do you guarantee that the default value is {}? It seems to be less efficient in the model than in the database itself.